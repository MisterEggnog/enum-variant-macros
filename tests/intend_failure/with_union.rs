use enum_variant_macros::*;
#[derive(FromVariants)]
union NotEnum {
	a: u32,
	b: f32,
}
fn main() {
}
