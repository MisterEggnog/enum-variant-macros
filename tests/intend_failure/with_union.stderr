error: Derive FromVariants for NotEnum failed. Must be an enum.
 --> tests/intend_failure/with_union.rs:3:1
  |
3 | union NotEnum {
  | ^^^^^
